overlord/ifacestate/ifacemgr.go

```diff
@@ -71,6 +71,7 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, extraInterfaces
 	runner.AddHandler("setup-profiles", m.doSetupProfiles, m.undoSetupProfiles)
 	runner.AddHandler("remove-profiles", m.doRemoveProfiles, m.doSetupProfiles)
 	runner.AddHandler("discard-conns", m.doDiscardConns, m.undoDiscardConns)
+	runner.AddHandler("auto-connect", m.doAutoConnect, m.undoAutoConnect)
```

Comment:
not super important, but auto-connect itself doesn't touch the disk or the security profiles, so in SetBlocked we could ignore it

Reply:
Done.